test_common/KokkosKernels_TestUtils.hpp

```diff
@@ -104,6 +139,22 @@ void EXPECT_NEAR_KK(Scalar1 val1, Scalar2 val2, Scalar3 tol,
   EXPECT_LE((double)AT1::abs(val1 - val2), (double)AT3::abs(tol)) << msg;
 }
 
+template <class Scalar1, class Scalar2, class Scalar3>
+void EXPECT_NEAR_KK_REL(Scalar1 val1, Scalar2 val2, Scalar3 tol,
```

Comment:
I think a very similar effect can be obtained by the following call:
```
EXPECT_NEAR_KK(val1, val2, Kokkos::max(val1, val2)*tol)
```

Reply:
Thanks, got rid of the extra function on https://github.com/kokkos/kokkos-kernels/pull/1433/commits/c05166de67c4421eab2a1f623fc796ebfa072506.